### ISSUE TITLE
PhotoPea doesn't work for importing pictures

### ISSUE THREAD

**Issue by LamprosPitsillos** [qutebrowser/qutebrowser#6622]
**Version info**:
```
qutebrowser v2.3.0
Git commit: 
Backend: QtWebEngine 5.15.5, based on Chromium 87.0.4280.144
Qt: 5.15.2
```

**Description**
While in [PhotoPea](https://www.photopea.com/) ,I cant import a file, drag n dropping works , but no UI appears to select a file from my file system. Tried in messager , it appears there.
Maybe its not a qutebrowser bug but the sites bug. Felt like reporting it. 

**How to reproduce**
Go to the aforementioned site , type <C-o> , or import using File->Import

**Comment by The-Compiler** [comment]
See [[QTBUG-92519] [REG 5.15.2 -> 5.15.3] Can't upload files on Photopea - Qt Bug Tracker](https://bugreports.qt.io/browse/QTBUG-92519)

**Comment by LamprosPitsillos** [comment]
Forgive me , I saw the issue that you linked but i don't see any comments or anything pointing to a fix.
What i understood was that its not qutebrowsers fault but Qt's. So we just wait for a Qt web engine update?
Thanks!

**Comment by The-Compiler** [comment]
Related changes (or in this case, the fix) are linked in the "Gerrit Reviews" sitebar: [Implement dialogs of File System Access API (Ib1624f80) · Gerrit Code Review](https://codereview.qt-project.org/c/qt/qtwebengine/+/354394)

The fix made it into Qt 6.2, which is [scheduled](https://wiki.qt.io/Qt_6.2_Release) for end of September - but note that it's a new major release, so qutebrowser will probably not be ready immediately once it's released (see #5395).
